File: creed-scala/src/main/scala/com/ethan/feature/Person.scala
package com.ethan.feature

/**
 * @description: spring-creed
 * @author: EthanCao
 * @email: ethan.caoq@foxmail.com
 * @date: 8/30/2022 5:40 PM
 */
trait Person {
  val name = "person"
  var age = 18
  def sayHello() = {
    println(s"Hello $name $age")
  }
  def eat() = {
    println(s" $name $age is eating")
  }
}

File: spring-creed-mvc/spring-creed-oauth2/spring-creed-oauth2-standalone/src/main/java/com/ethan/std/vo/Person.java
package com.ethan.std.vo;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Data;

/**
 * @description: spring-creed
 * @author: EthanCao
 * @email: ethan.caoq@foxmail.com
 * @date: 8/16/2022 11:31 AM
 */
@Data
@JsonInclude(JsonInclude.Include.NON_EMPTY)
public class Person {
    private String name;
    private String title;
}
